lib/RequestLogger.js

```diff
@@ -222,6 +222,19 @@ class RequestLogger {
     }
 
     /**
+     *  This function creates a new RequestLogger object from a ModuleLogger
+     *  already existing in order to have the submodule field from the
+     *  ModuleLogger.
+     *
+     *	@param {ModuleLogger} ml - The base ModuleLogger
+     *	@return {RequestLogger} - The new RequestLogger created
+     *				  with the ModuleLogger
+     */
+    scope(ml) {
```

Comment:
Because in fact the ScopedRequestLogger is exactly the same as a RequestLogger with only the submodule differing, but present in all cases, so the ScopedRequestLogger class will just be a kind of copy/paste of RequestLogger. So I thought creating a new RequestLogger using the right ModuleLogger was enough and simpler.